README.md

```diff
@@ -1,64 +1,5 @@
 # Tests for Mozilla's Treeherder
-This repository contains tests for [Mozilla's Treeherder](https://treeherder.mozilla.org/).
+This repository holds automation code that was used for Mozilla's Treeherder website. All code has been moved to the [primary Treeherder repo](https://github.com/mozilla/treeherder).
```

Comment:
@rbillings nah, it's totally a nit.  Merge away after conflict resolution + final-green-button enablement!